internal/services/compute/orchestrated_virtual_machine_scale_set_resource_disk_os_test.go

```diff
@@ -30,6 +30,13 @@ func TestAccOrchestratedVirtualMachineScaleSet_disksOSDiskCaching(t *testing.T)
 			),
 		},
 		data.ImportStep("os_profile.0.linux_configuration.0.admin_password"),
+		{
+			Config: r.disksOSDiskEphemeral(data, "NvmeDisk"),
```

Comment:
Rather than go this direction, you can make a new test to specifically build this with an `NvmeDisk`.